app/redux/actions/async.js

```diff
@@ -2384,17 +2384,17 @@ export function fetchPatientInvites(api, clinicId) {
  * @param {String} clinicId - Id of the clinic
  * @param {String} inviteId - Id of the invite
  */
-export function acceptPatientInvitation(api, clinicId, inviteId) {
+export function acceptPatientInvitation(api, clinicId, inviteId, patientId, patientDetails) {
```

Comment:
Should probably add param details to the docblock.

Reply:
Will do.